scripts/esp32_log_cat.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
```

Comment:
Are you aware of

```
source examples/all-clusters-app/esp32/idf.sh      
export IDF_PATH=${HOME}/tools/esp-idf
idf make -C examples/all-clusters-app/esp32 monitor
```

?

If this is needed can you move it to a different PR?

Reply:
Removed.

I am aware of monitor, however I generally build from vscode and monitor does not seem to allow as convenient filtering (like show warning and above only) nor does it show things as pretty.